src/migrate/index.js

```diff
@@ -119,10 +166,22 @@ export default class Migrator {
       .then((batchNo) => {
         return this._waterfallBatch(batchNo, migrations, direction)
       })
+      .then(() => this._unlockMigrations())
       .catch((error) => {
         helpers.warn('migrations failed with error: ' + error.message)
-        throw error
-      })
+
+        // If the error was not due to a locking issue, then
+        // remove the lock. Otherwise, leave it in place to
+        // force manual intervention
+        if (error.message !== 'migrations failed: migration in progress') {
```

Comment:
And here checking by error type instead of error message would be cleaner (changing message above would break this otherwise...).